cabby/data/wikipedia/extract_wikipedia_items.py

```diff
@@ -0,0 +1,45 @@
+# coding=utf-8
+# Copyright 2020 Google LLC
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+'''Example command line method to extract Wikipedia items.
+Example:
+$ bazel-bin/cabby/data/wikipedia/extract_wikipedia_items --titles=New_York_Stock_Exchange, Empire_State_Building
```

Comment:
Please make sure your commands work by cutting-and-pasting: this needs quotes around the titles. Also good to keep it <80 chars, so more like this:

$ bazel-bin/cabby/data/wikipedia/extract_wikipedia_items \
   --titles="New_York_Stock_Exchange, Empire_State_Building"